### PR TITLE
Add recipe for awscli-cwlogs

### LINKED ISSUE #6085: Package awscli-cwlogs
It would be nice to have a package for [awscli-logs](https://pypi.org/project/awscli-cwlogs/) an awscli plugin to provide pull, push and filter commands to access AWS CloudWatch Logs service.

cf: #6084

### PR DESCRIPTION
**Pull request by CurtLH**
Closes #6085

### CODE REVIEW

**Comment by CurtLH** [comment]
The `LICENSE.txt` for [`awscli-cwlogs`](https://pypi.org/project/awscli-cwlogs/) is an ["Amazon Software License"](https://aws.amazon.com/asl/).  According to the license field on PyPI, is says that this is "OSI Approved".  I searched Conda-Forge for other packages mentioning "Amazon Software License", but didn't come across anything else.  

How should I list the license in this package @conda-forge/staged-recipes?

File: recipes/awscli-cwlogs/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    - six >=1.1.0
+    - python-dateutil >=2.1
+
+test:
+  imports:
+    - cwlogs
+
+about:
+  home: http://aws.amazon.com/cli/
+  license: Apache-2.0
```
[the remark this answers is not in the document]
Comment (scopatz) [reply recipes/awscli-cwlogs/meta.yaml]: It actually does seem to be apache: https://github.com/aws/aws-cli/blob/develop/LICENSE.txt

[the remark this answers is not in the document]
Comment (mariusvniekerk) [reply recipes/awscli-cwlogs/meta.yaml]: Yeah, just done oddly

**Comment by sodre** [comment]
Thanks @CurtLH for building it, and @mariusvniekerk and @scopatz  for reviewing and merging!

**Pull request merged**